docs/delivery-manual/align.md

```diff
@@ -19,37 +18,37 @@ The Align Phase initiates the digital trust adoption journey by creating awarene
 ## Core Activities
 Core Activities are key tasks to complete during the Align phase, either independently or with Digital Trust team support, to prepare for adoption.
 
-| | Ideation Workshop (Activity)| 
-| ------ |  :--------  |
-| ![A light bulb emoji](/img/deliverymanual/lightbulb.png)| An initial workshop focused on brainstorming potential use cases for digital credentials. |
-| ![A clock emoji](/img/deliverymanual/time.png)|60+ minutes | 
-| ![An emoji of two figures hugging](/img/deliverymanual/participants.png)| Project team | 
-| ![A briefcase emoji](/img/deliverymanual/tool.png)| Miro |
-| ![An arrow pointing left emoji](/img/deliverymanual/arrow_left.png) | Ideation Workshop |
----
-| | Ecosystem Workshop (Activity)| 
-| ------ |  :--------  |
-| ![A light bulb emoji](/img/deliverymanual/lightbulb.png)| Helps understand and visualize how your digital trust use case fits within an ecosystem. |
-| ![A clock emoji](/img/deliverymanual/time.png)|60+ minutes | 
-| ![An emoji of two figures hugging](/img/deliverymanual/participants.png)| Project team | 
-| ![A briefcase emoji](/img/deliverymanual/tool.png)| Miro |
-| ![An arrow pointing left emoji](/img/deliverymanual/arrow_left.png) | Ecosystem Workshop|
----
-| | Skills and Resource Assessment Workshop (Activity)| 
-| ------ |  :--------  |
-| ![A light bulb emoji](/img/deliverymanual/lightbulb.png)| An evaluation to determine the necessary skills and resources for the use case. |
-| ![A clock emoji](/img/deliverymanual/time.png)|60+ minutes | 
-| ![An emoji of two figures hugging](/img/deliverymanual/participants.png)| Project team | 
-| ![A briefcase emoji](/img/deliverymanual/tool.png)| Miro |
-| ![An arrow pointing left emoji](/img/deliverymanual/arrow_left.png) | Skills and Resource Assessment |
----
-|  | Readiness Checklist (Document)|
-| ------ |  :--------  | 
-| ![A light bulb emoji](/img/deliverymanual/lightbulb.png) | A tool designed to assess the adopter’s readiness to progress to the next phase. | 
-| ![A clock emoji](/img/deliverymanual/time.png)| 45+ minutes | 
-| ![An emoji of two figures hugging](/img/deliverymanual/participants.png)| Project team leads | 
-| ![A briefcase emoji](/img/deliverymanual/tool.png)| Miro |
-| ![An arrow pointing left emoji](/img/deliverymanual/arrow_left.png) | Readiness Checklist |
+### Ideation Workshop
+==Activity==
+Creating a flowchart to detail the interactions and steps involving each actor in the use case.
```

Comment:
Wrong content: Should be "An initial workshop focused on brainstorming potential use cases for digital credentials."